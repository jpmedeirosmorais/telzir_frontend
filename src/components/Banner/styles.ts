import styled from "styled-components";
import { Colors } from "common/styles";

export const Content = styled.div`
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  color: ${Colors.light.yellow};
`;

export const Title = styled.div`
  font-size: 4rem;
  font-weight: bold;
  margin-bottom: 5rem;
  width: 100vw;
  background-color: ${Colors.light.yellow};
  color: ${Colors.dark.grey};
  padding: 1.5rem 5rem;
`;

export const Description = styled.div`
  color: ${Colors.light.yellow};
  text-align: center;
  font-size: 2rem;
  font-weight: bold;
  flex-wrap: nowrap;
`;

export const Text = styled.div`
  max-width: 70rem;
  margin-left: auto;
  margin-right: auto;
`;
