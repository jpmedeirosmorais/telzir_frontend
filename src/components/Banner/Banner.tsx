import * as S from "./styles";
import { Logo } from "common/images";

export const Banner = () => {
  return (
    <S.Content>
      <S.Title>
        Telzir
        <Logo />
      </S.Title>
      <S.Description>
        <S.Text>
          Bem vindo a Telzir! Conheça agora nossos novos planos de chamada a
          longa distância. Veja como você pode aproveitar todos os benefícios.
          Temos os planos FaleMais 30, FaleMais 60 e FaleMais 120. Faça a
          simulação agora mesmo inserindo os dados abaixo e veja qual plano se
          encaixa melhor ao seu perfil.
        </S.Text>
      </S.Description>
    </S.Content>
  );
};
