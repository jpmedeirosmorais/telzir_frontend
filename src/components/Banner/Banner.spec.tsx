import { render } from "@testing-library/react";

import { Banner } from "./Banner";

describe("Banner", () => {
  it("renders", () => {
    expect(render(<Banner />)).toMatchSnapshot();
  });
});
