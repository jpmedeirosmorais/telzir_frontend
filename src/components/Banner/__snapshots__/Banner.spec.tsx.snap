// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Banner renders 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="sc-bczRLJ bRbpkm"
      >
        <div
          class="sc-gsnTZi fRUUuX"
        >
          Telzir
          <svg>
            call_black.svg
          </svg>
        </div>
        <div
          class="sc-dkzDqf fkkMXm"
        >
          <div
            class="sc-hKMtZM jBCljQ"
          >
            Bem vindo a Telzir! Conheça agora nossos novos planos de chamada a longa distância. Veja como você pode aproveitar todos os benefícios. Temos os planos FaleMais 30, FaleMais 60 e FaleMais 120. Faça a simulação agora mesmo inserindo os dados abaixo e veja qual plano se encaixa melhor ao seu perfil.
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="sc-bczRLJ bRbpkm"
    >
      <div
        class="sc-gsnTZi fRUUuX"
      >
        Telzir
        <svg>
          call_black.svg
        </svg>
      </div>
      <div
        class="sc-dkzDqf fkkMXm"
      >
        <div
          class="sc-hKMtZM jBCljQ"
        >
          Bem vindo a Telzir! Conheça agora nossos novos planos de chamada a longa distância. Veja como você pode aproveitar todos os benefícios. Temos os planos FaleMais 30, FaleMais 60 e FaleMais 120. Faça a simulação agora mesmo inserindo os dados abaixo e veja qual plano se encaixa melhor ao seu perfil.
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
